packages/core/crates/core-protocol/src/msg/processor.rs

```diff
@@ -74,7 +73,7 @@ const PACKET_RX_QUEUE_SIZE: usize = 2048;
 #[derive(Debug)]
 pub enum MsgToProcess {
     ToReceive(Box<[u8]>, PeerId),
-    ToSend(ApplicationData, Path, PacketSendFinalizer),
+    ToSend(ApplicationData, TransportPath, PacketSendFinalizer),
```

Comment:
The term `TransportPath` is slightly misleading though, because the physical transport path is not part of the weight, so a path that is deemed OK might now eventually be a successful one. Maybe a better name would be `ChannelPath`.

Reply:
So `ChannelPath` is the type that describes the path in the `ChannelGraph`. From that you can construct a `TransportPath` which has an additional entry and that's the packet destination peer ID. 
What naming would you suggest?

Reply:
Maybe we can leave it, bit still, it's misleading a little bit.